release/models/isis/openconfig-isis-policy.yang

```diff
@@ -200,15 +206,19 @@ module openconfig-isis-policy {
       description
         "Formerly this leaf sets the type of metric that is to be specified
         when the set-metric leaf is specified.  This leaf is DEPRECATED
-        due to being assigned the wrong type. Use set-metric-style-type
-        instead.";
+        due to being assigned the wrong type. Use set-route-type instead.";
+    }
+
+    leaf set-route-type {
```

Comment:
This is setting metric type, right?  Since this is already a breaking change, due to renaming `set-metric-type` leaf   we should change the type for `set-metric-type`.   

    leaf set-metric-type {
      type isis-types:metric-type;
      description
        "This leaf sets the type of metric.";
    }

The ISIS policy model is a version 0.x model, which we permit breaking changes without a major OpenConfig release.  In this case the intent is to change names to align with the semantics of "match", " action" and "type"s being used, I feel like we should go proceed with the breaking change.

Reply:
modified to proceed with breaking change